modules/library/referencing/src/main/java/org/geotools/referencing/cs/DefaultCoordinateSystemAxis.java

```diff
@@ -523,7 +523,7 @@
      * A default axis for time values in a {@linkplain org.opengis.referencing.cs.TimeCS time CS}.
      *
      * <p>Increasing time go toward {@linkplain AxisDirection#FUTURE future} and units are
-     * {@linkplain NonSI#DAY days}.
```

Comment:
Here is an example of a NonSI type that became an SI unit.

Reply:
Gotcha!